site/book/01-getting-started/00.md

```diff
@@ -0,0 +1 @@
+# Getting Started
```

Comment:
Can you auto-generate the title from the directory and file names?

Reply:
We can if we want to keep URLs like `/book/01-getting-started/01-quickstart` but not if we switch to `/book/01-getting-started/01` (since URLs are determined by the markdown path).

Reply:
Added title auto-generation.